systools/serial_log_decoder/serial_log_decoder.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+'''
+Created on 08/11/2011
+
+@author: aes
+
+Original version:
+https://stash.silabs.com/projects/UIC/repos/uic/commits/c95879e16432600fc20897b9b90b909ee268c794#scripts/serial_decode_zpc.py
```

Comment:
Unsure we want to keep reference to non public place

Reply:
Done. Please, resolve if OK.